packages/backend/src/core/discovery/utils/diffToString.ts

```diff
@@ -0,0 +1,32 @@
+import { DiscoveryDiff } from './diffDiscovery'
+
+export function diffToString(diff: DiscoveryDiff): string {
+  if (diff.type === 'created') {
+    return `\`\`\`diff\n+New contract: ${
+      diff.name
+    } | ${diff.address.toString()}\n\`\`\``
+  }
```

Comment:
Once this becomes `diffToMessage` individual changes can be part of a single code block instead of requiring their own code blocks